include/behaviortree_cpp/bt_factory.h

```diff
@@ -44,13 +44,20 @@ template <typename T>
 inline TreeNodeManifest CreateManifest(const std::string& ID,
                                        PortsList portlist = getProvidedPorts<T>())
 {
-  if constexpr( has_static_method_description<T>::value)
+  if constexpr( has_static_method_description<T>::value && has_static_method_metadata<T>::value )
```

Comment:
I think from our last discussion with @facontidavide we said we'd just replace description with metadata instead of adding it, right?

Where the description would just be one element of the metadata, if users wanted it?

Reply:
yes

Reply:
I replaced `description` with `metadata` in https://github.com/BehaviorTree/BehaviorTree.CPP/pull/730/commits/96910f265786458ac2f1b33a9f3d1a3adaf26473